test/unittests/xmlParse/xmlRegisterContextRequest_test.cpp

```diff
@@ -1,142 +0,0 @@
-  EXPECT_EQ(expected, out);
-
-  utExit();
-}
-
-
-
-/* ****************************************************************************
-*
-* invalidSourceEntityId - 
-*/
-TEST(xmlRegisterContextRequest, invalidSourceEntityId)
-{
-  ConnectionInfo  ci("/ngsi9/registerContext", "POST", "1.1");
-  const char*     inFile = "ngsi9.registerContextRequest.invalidSourceEntityId.invalid.xml"; 
-  const char*     outFile1 = "ngsi9.registerContextResponse.invalidSourceEntityId.valid.xml";
-  ParseData       parseData;
-  std::string     out;
-
-  utInit();
-
-  EXPECT_EQ("OK", testDataFromFile(testBuf, sizeof(testBuf), inFile)) << "Error getting test data from '" << inFile << "'";
-  EXPECT_EQ("OK", testDataFromFile(expectedBuf, sizeof(expectedBuf), outFile1)) << "Error getting test data from '" << outFile1 << "'";
-
-  ci.inFormat  = XML;
-  ci.outFormat = XML;
-  out  = xmlTreat(testBuf, &ci, &parseData, RegisterContext, "registerContextRequest", NULL);
-  EXPECT_STREQ(expectedBuf, out.c_str());
-
-  utExit();
-}
-
-
-
-/* ****************************************************************************
-*
-* invalidTargetEntityId - 
-*/
-TEST(xmlRegisterContextRequest, invalidTargetEntityId)
-{
-  ConnectionInfo  ci("/ngsi9/registerContext", "POST", "1.1");
-  const char*     inFile  = "ngsi9.registerContextRequest.invalidTargetEntityId.invalid.xml"; 
-  const char*     outFile = "ngsi9.registerContextResponse.invalidTargetEntityId.valid.xml";
-  ParseData       parseData;
-  std::string     out;
-
-  utInit();
-
-  EXPECT_EQ("OK", testDataFromFile(testBuf, sizeof(testBuf), inFile)) << "Error getting test data from '" << inFile << "'";
-  EXPECT_EQ("OK", testDataFromFile(expectedBuf, sizeof(expectedBuf), outFile)) << "Error getting test data from '" << outFile << "'";
-
-  ci.inFormat  = XML;
-  ci.outFormat = XML;
-  out  = xmlTreat(testBuf, &ci, &parseData, RegisterContext, "registerContextRequest", NULL);
-  EXPECT_STREQ(expectedBuf, out.c_str());
-
-  utExit();
-}
-
-
-
-/* ****************************************************************************
-*
-* twoEntityIdLists - 
-*/
-TEST(xmlRegisterContextRequest, twoEntityIdLists)
-{
-  ConnectionInfo  ci("/ngsi9/registerContext", "POST", "1.1");
-  const char*     inFile  = "ngsi9.registerContextRequest.twoEntityIdLists.invalid.xml"; 
-  const char*     outFile = "ngsi9.registerContextResponse.twoEntityIdLists.valid.xml"; 
-  ParseData       parseData;
-  std::string     out;
-
-  utInit();
-
-  EXPECT_EQ("OK", testDataFromFile(testBuf, sizeof(testBuf), inFile)) << "Error getting test data from '" << inFile << "'";
-  EXPECT_EQ("OK", testDataFromFile(expectedBuf, sizeof(expectedBuf), outFile)) << "Error getting test data from '" << outFile << "'";
-
-  ci.inFormat  = XML;
-  ci.outFormat = XML;
-  out  = xmlTreat(testBuf, &ci, &parseData, RegisterContext, "registerContextRequest", NULL);
-  EXPECT_STREQ(expectedBuf, out.c_str());
-
-  utExit();
-}
```

Comment:
So much work, to later after almost never using it, throw it away ...
Very good to cleanup our source code, of course!

NTC